### DISCUSSION TITLE
怎麼申請護照

### DISCUSSION THREAD

**Discussion by zhilin93517** [1101-GS4719-C/Python-Community-GS4719-C#2142] [General]
最近想去辦一個護照，查了一下護照的辦理流程，整理了一下，換發護照的流程如下:

1. 不是所有縣市都可以換發護照，要到特定縣市才可以，像是台北或嘉義
2. 可以使用預約的方式排隊，免去現場排隊所花費的時間
3. 應攜帶大頭照兩張
4. 應攜帶身分證
5. 應攜帶彩色列印之簡式護照資料表正本
6. 尚有效期之舊護照

我有曾經試著寄信給局長問問題，但局長並沒有回覆，真的謝了。

**Comment by Jing-Asteroid** [comment]
我上次去局長家登門拜訪他就有回覆我ㄟ，會不會是因為你的方式不對啊，或是態度不夠端莊?
局長人也很好，上去空手去拜訪他，結束後雙手都有手銬做禮物還有專人陪行離開，離開還有座駕，看車身黑白相間的也是別有一番藝術氣息。

**Comment by NiCL100** [comment]
只有到特定縣市才能申換護照真是過分！這使得某些縣市的人民需要特地花錢花時間至特定縣市，十分不便民。雖然不了解公務機關的作業流程及限制，但我認為他們應該以「各縣市皆可辦理申換護照」為目標，逐步使得人民的生活更加便利。

**Comment by zhilin93517** [comment]
更正，局長方有打電話給我確認過我的需求了，他們有依照我的需求提供協助，而且態度十分親合，處理速度也相當得快，非常感謝。我覺得他們在這方面的答覆是有求必應，有幫助到我。若有找不到相關問題的答覆可以嘗試寫信給局長看看。

**Comment by zhilin93517** [comment]
我上次問的問題是，若我申請方式選擇的是親自辦理換發護照，在領取護照時是否可以選擇請他人代領，若可以，是否需要提供相關證件。局長給的回覆是可以請他人代領，僅須提供辦理時的收據即可領取護照。我覺得官方回復得很清楚，也很照顧到我是否可以理解他的說法，我對這次的體驗感到很滿意。

**Comment by Jinyuu17** [comment]
許多旅行社都提供護照代辦服務，這對於第一次辦護照或不想自己處理繁瑣手續的人來說非常便利。只需要多準備委任書和代辦費。更多詳細的內容可以參考旅行社網站或外交部官網。
https://www.boca.gov.tw/cp-35-6836-66374-1.html

**Comment by zhilin93517** [comment]
此外，我發現到領取護照時，除了可以親自領取、請他人代領之外，櫃檯旁邊的中華郵政也可以協助領取，並且將領取的護照直接寄到指定地址，對於這點我覺得非常方便，他彌補了非各縣市皆有可以辦領護照的據點的缺點，讓資源能夠更集中，我也因此解決了我問題。
